alembic/versions/bc0f6ee73781_initial_migration.py

```diff
@@ -0,0 +1,30 @@
+"""Initial migration
+
+Revision ID: bc0f6ee73781
+Revises: 
+Create Date: 2024-11-07 09:43:55.713176
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = 'bc0f6ee73781'
+down_revision: Union[str, None] = None
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    pass
+    # ### end Alembic commands ###
+
+
+def downgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    pass
```

Comment:
The `downgrade` function is also empty. If you need to support downgrading the schema, you should add the corresponding operations here.